packages/flutter/test/material/dropdown_test.dart

```diff
@@ -2949,4 +2949,57 @@ void main() {
     expect(scrollController.position.maxScrollExtent > 0, isTrue);
     expect(find.byType(Scrollbar), paints..rect());
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/76614
+  testWidgets('Do not crash if used in very short screen', (WidgetTester tester) async {
+    const String value = 'foo';
+    final UniqueKey itemKey = UniqueKey();
+    await tester.pumpWidget(
+      Center(
+        child: SizedBox(
```

Comment:
I wonder if we should do something similar to https://stackoverflow.com/questions/53706569/how-to-test-flutter-widgets-on-different-screen-sizes, where we adjust the physical screen size and the device pixel ratio. Perhaps that would emulate a real-world scenario better?

Reply:
Nice!
I did not know this API before!